cmd/test_runner_test.go

```diff
@@ -421,25 +420,30 @@ func (run *TestRunner) LookupVersionInfo(t *testing.T, refStr string) *dsref.Ver
 }
 
 // ClearFSIPath clears the FSIPath for a reference in the refstore
+// Note: ClearFSIPAth doesn't do reference resoultion, cannot use "me" in
```

Comment:
ClearFSIPAth -> ClearFSIPath, resoultion -> resolution. Also, big fan of this change and the clarifying comment.